pages/CompressPage.py

```diff
@@ -16,6 +19,11 @@ class CompressPage(tk.Frame):
 
     def __init__(self, parent, controller):
         tk.Frame.__init__(self, parent)
+
+        self.transcoder = Transcoder()
+        self.save_path = ""
```

Comment:
Changing these two since using global variable somehow does not work (even though the user has given input for the paths, the changes do not reflect into the `transcoder.transcode()` function.

Reply:
Approved.